types/src/v0/impls/block/full_payload/payload.rs

```diff
@@ -219,14 +219,6 @@ impl QueryablePayload<SeqTypes> for Payload {
 
         TxProof::new(index, self, &common)
     }
-
-    fn transaction(
```

Comment:
why are we removing this?

Reply:
oh we shouldn't, I tested with this change, just didn't push it, I'll fix it